specification/policyinsights/resource-manager/readme.go.md

```diff
@@ -23,6 +23,7 @@ modelerfour:
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2023-03
```

Comment:
We do not need to add this any more, because this is only for the track 1 Go SDK which is officially announced as deprecated.
Could we revert changes in this file?